jobs/furnishings/src/furnishings/config.py

```diff
@@ -87,7 +87,15 @@ class _Config:  # pylint: disable=too-few-public-methods
 
     SECOND_NOTICE_DELAY = int(os.getenv('SECOND_NOTICE_DELAY', '5'))
     LEGISLATIVE_TIMEZONE = os.getenv('LEGISLATIVE_TIMEZONE', 'America/Vancouver')
-    TEMPLATE_PATH = os.getenv('TEMPLATE_PATH', 'src/furnishings-templates')
+    XML_TEMPLATE_PATH = os.getenv('XML_TEMPLATE_PATH', 'furnishings-templates')
```

Comment:
I renamed to distinguish it with letter templates, and also moved it to the same directory of letter templates. Let me know if I need to revert some changes here.